--- /tmp/psplash/psplash.c	2010-08-26 20:29:38.380562430 +0100
+++ psplash/psplash.c	2010-08-26 21:20:05.410436911 +0100
@@ -22,6 +22,7 @@
 #include "psplash-hand-img.h"
 #include "psplash-bar-img.h"
 #include "radeon-font.h"
+#include "psplash-config.h"
 
 #define MSG ""
 
@@ -49,7 +50,7 @@
 			fb->height - (fb->height/6) - h, 
 			fb->width,
 			h,
-			0xec, 0xec, 0xe1);
+			BG_R, BG_G, BG_B);
 
   psplash_fb_draw_text (fb,
 			(fb->width-w)/2, 
@@ -75,19 +76,19 @@
       barwidth = (CLAMP(value,0,100) * width) / 100;
       psplash_fb_draw_rect (fb, x + barwidth, y, 
     			width - barwidth, height,
-			0xec, 0xec, 0xe1);
+			BARB_R, BARB_G, BARB_B);
       psplash_fb_draw_rect (fb, x, y, barwidth,
-			    height, 0x6d, 0x6d, 0x70);
+			    height, BARF_R, BARF_G, BARF_B);
     }
   else
     {
       barwidth = (CLAMP(-value,0,100) * width) / 100;
       psplash_fb_draw_rect (fb, x, y, 
     			width - barwidth, height,
-			0xec, 0xec, 0xe1);
+			BARB_R, BARB_G, BARB_B);
       psplash_fb_draw_rect (fb, x + width - barwidth,
 			    y, barwidth, height,
-			    0x6d, 0x6d, 0x70);
+			    BARF_R, BARF_G, BARF_B);
     }
 
   DBG("value: %i, width: %i, barwidth :%i\n", value, 
@@ -252,8 +253,8 @@
   if ((fb = psplash_fb_new(angle)) == NULL)
     exit(-1);
 
-  /* Clear the background with #ecece1 */
-  psplash_fb_draw_rect (fb, 0, 0, fb->width, fb->height, 0xec, 0xec, 0xe1);
+  /* Clear the background with the background colour */
+  psplash_fb_draw_rect (fb, 0, 0, fb->width, fb->height, BG_R, BG_G, BG_B);
 
   /* Draw the OH logo  */
   psplash_fb_draw_image (fb, 
