From 54027c1e0ce612a04b89866531db76416746cc3b Mon Sep 17 00:00:00 2001
From: Robert Spanton <rspanton@zepler.net>
Date: Fri, 23 Sep 2011 15:35:57 +0100
Subject: [PATCH 1/2] Disable the display

---
 board/ti/beagle/beagle.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/board/ti/beagle/beagle.c b/board/ti/beagle/beagle.c
index aa5047c..bdbdcd5 100644
--- a/board/ti/beagle/beagle.c
+++ b/board/ti/beagle/beagle.c
@@ -354,7 +354,7 @@ int misc_init_r(void)
 		GPIO15 | GPIO14 | GPIO13 | GPIO12), &gpio5_base->oe);
 
 	dieid_num_r();
-	beagle_display_init();
+	/* beagle_display_init(); */
 	omap3_dss_enable();
 
 	return 0;
-- 
1.7.6.2

